plugins/woocommerce-admin/client/store-management-links/style.scss

```diff
@@ -3,3 +3,7 @@
 .woocommerce-store-management-links__card-body.is-size-custom {
 	padding: 24px 0 8px 0;
 }
+
+.woocommerce-store-management-links__card-body {
+	padding-top: 24px;
```

Comment:
Good eye! @ilyasfoo What do you think if we use `padding: $gap-large 0 $gap-smaller 0;` (`padding: 24px 0 8px 0;`)?

It looks like it was supposed to be like that.

https://github.com/woocommerce/woocommerce/blob/92a18587311d0f9f3494e794b1383bff7342e2c4/plugins/woocommerce-admin/client/store-management-links/style.scss#L2-L5

Reply:
Just made the changes first in 30aab39f8510c0428efa92cbab95bcb363a46fe4!

Reply:
Good catch, however, I'm seeing the padding that's added gives a "border-like" artefact at the bottom when hovering the last item:

<img width="524" alt="image" src="https://user-images.githubusercontent.com/3747241/237000539-084212f7-a435-4275-bdff-b7e47029962b.png">
<img width="524" alt="image" src="https://user-images.githubusercontent.com/3747241/237001186-44009323-2e7d-49fa-a7c0-74cfdbd7d955.png">

Whereas, in inbox notes items, tasklist items do not have this.

But perhaps I may be overly complicating it. I don't know if we need to call our expert @verofasulo for opinion 🙏 😄

Reply:
> I'm seeing the padding that's added gives a "border-like" artefact at the bottom when hovering the last item

Good catch, @ilyasfoo! I agree that it would be better to remove the padding there to avoid that effect.

Reply:
Thanks @verofasulo @ilyasfoo! Updated as suggested in e4af20ddda7ff894054dae9ff84d01d4af7f6c8a. 🙌